python/ray/tests/test_object_store_metrics.py

```diff
@@ -220,43 +220,6 @@ def test_spilling(object_spilling_config, shutdown_only):
     )
 
 
-@pytest.mark.skipif(
-    sys.platform == "darwin", reason="Timing out on macos. Not enough time to run."
-)
-@pytest.mark.parametrize("metric_report_interval_ms", [500, 1000, 3000])
```

Comment:
@rickyyx I believe this test is not necessary. (I think this should be already tested by other test like test_metrics.py). Is it okay to just remove it?

Reply:
yeah - i feel we probably don't need to run tests on each metric?

Reply:
Or if there's an easier way we could do it instead of individual tests